data/lists/brr.csv

```diff
@@ -0,0 +1,27 @@
+# Instances used in "Achterberg, T., Koch, T., & Martin, A. (2005). Branching rules revisited. Operations Research Letters, 33(1), 42-54."
```

Comment:
Why is the name brr? is it because it is imported from the one used by the authors?, what about   we use a more descriptive name like `achterberg.csv` and optionally we can add subdirectory `miplib`.

Reply:
Was for branching_rules_revisited. I somehow feel like the title of the paper here is more descriptive then the first author. Open for discussion.

Reply:
Yes, I agree.